### DISCUSSION TITLE
Why is the Docker image of ollama-webui so large?

### DISCUSSION THREAD

**Discussion by dgreene423** [open-webui/open-webui#420] [Q&A]
Maybe it's just me, but this seems way larger than necessarily.  

❯ docker images ghcr.io/ollama-webui/ollama-webui
REPOSITORY                          TAG       IMAGE ID       CREATED       SIZE
ghcr.io/ollama-webui/ollama-webui   main      78ee07b66376   6 hours ago   8.46GB

**Comment by tjbck** [comment]
Related #415

**Comment by dgreene423** [comment]
thanks, @tjbck
